code/modules/surgery/bodyparts/head.dm

```diff
@@ -141,6 +141,7 @@
 	else if(!animal_origin)
 		var/mob/living/carbon/human/H = C
 		var/datum/species/S = H.dna.species
+		H.checknoosedrop()
```

Comment:
Why are you calling this in update_limb()?

Reply:
not exactly sure, was direct ported from hippiecode, ill look into it, any suggestions here?

Reply:
>removed this and the proc itself as its not needed it seems